src/tool.rs

```diff
@@ -213,15 +214,15 @@ impl Tool {
                 Some(fname) if fname.contains("clang") => match clang_driver {
                     Some("cl") => ToolFamily::Msvc { clang_cl: true },
                     _ => ToolFamily::Clang {
-                        zig_cc: is_zig_cc(&path, cargo_output),
+                        zig_cc: is_zig_cc(&path, cargo_output).unwrap_or(false),
```

Comment:
`unwrap_or(false)` here is a bit janky. I don't think it makes much of a difference what we report when `cc` is disabled, though.